ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -403,7 +408,7 @@ public void init() {
           String revertId = req.params(":revertid");
           String storeId = req.params(":storeId");
           Metacard versionMetacard = revert(id, revertId, storeId);
-          return util.metacardToJson(MetacardVersionImpl.toMetacard(versionMetacard, types));
```

Comment:
Just removing an argument that is no longer used. The old form of toMetacard is deprecated and the "types" argument is now ignored.